hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -213,11 +213,13 @@ def _update_calendar_event(self):
             if rec.meeting_id:
                 rec.meeting_id.write(rec._prepare_holidays_meeting_values())
 
-    @api.model
+    @api.model_create_multi
+    @api.returns('self', lambda value: value.id)
```

Comment:
I don't think you need to indicate this return decorator

Reply:
indeed... leftover from old times :)